src/patternfly/components/Breadcrumb/breadcrumb-link.hbs

```diff
@@ -0,0 +1,6 @@
+<a href="{{breadcrumb-link--href}}" class="pf-c-breadcrumb__link{{#if breadcrumb-link--modifier}} {{breadcrumb-link--modifier}}{{/if}}"
```

Comment:
For the sake of this example I think it's ok to just have `href="#"` rather than setting it every time in the examples. @michael-coker what do you think?

Reply:
Yep, agreed! I doubt we'll ever need to set this to anything but `#` in the workspace.